e2e/serial/swap/1_swapFlow1.test.ts

```diff
@@ -445,7 +446,10 @@ it('should be able to check price and balance of token to buy', async () => {
   expect(tokenToBuyInfoBalance).not.toBe('');
 });
 
-it('should be able to flip correctly', async () => {
+// we fixed a preexisting issue with output swaps, which breaks
```

Comment:
need to figure out if we want this test still with the output changes and if so, redo it.